WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -121,16 +135,43 @@ class ReaderPostCardActionsHandler @Inject constructor(
         _navigationEvents.postValue(Event(OpenPost(post)))
     }
 
-    private fun handleBlockSiteClicked(postId: Long, blogId: Long) {
-        AppLog.d(AppLog.T.READER, "Block site not implemented")
+    private fun handleBlockSiteClicked(blogId: Long) {
```

Comment:
As mentioned in the description `ReaderPostCardActionsHandler` is responsible for transforming the state into a UI action.